src/main/java/com/mateacademy/patterny/abstractFactory/AuctionSiteProject.java

```diff
@@ -0,0 +1,18 @@
+package com.mateacademy.patterny.abstractFactory;
+
+import com.mateacademy.patterny.abstractFactory.website.WebsiteTeamFactory;
+
+public class AuctionSiteProject {
+    public static void main(String[] args) {
+        ProjectTeamFactory projectTeamFactory = new WebsiteTeamFactory();
```

Comment:
You use concretic factory|class. It is hardcode. Read https://www.baeldung.com/java-abstract-factory-pattern

Reply:
I add new commit: commit 21.02.2019 - abstrack factory 2 where i fixed

Reply:
Ok. I see there same as before.